doc/btp_gap.txt

```diff
@@ -891,3 +911,18 @@ Events:
 
                 This event can be sent after IUT received encryption change event was received.
                 The parameter `Enabled` is the flag whether the link is encrypted.
+
+        Opcode 0x92 - Subrate change event
+
+                Controller Index:       <controller id>
+                Event parameters:       Address_Type (1 octet)
+                                        Address (6 octets)
+                                        Status (1 octet)
+                                        Conn_Handle (2 octets)
+                                        Subrate_Factor (2 octets)
+                                        Peripheral_Latency (2 octets)
+                                        Continuation_number (2 octets)
+                                        Supervision_Timeout (2 octets)
+
```

Comment:
we should start documenting those better,   however this can be done in followup PR